### ISSUE TITLE
Discord?

### ISSUE THREAD

**Issue by Decme** [mbnuqw/sidebery#326]
Is there a support discord? I have some questions. I'm looking to install this addon to help with the tab overload, problem is, I know that some addons 'reload' all tabs that they need to 'move', be it to a different container, or window.

But I have over 6000 tabs open.

So you can imagine, I wouldn't want all youtube tabs for example to reload at the same time. Whether it's 300 youtube tabs or 2000, the same end result is probably a crash and possible data loss, be it youtube or any other site.

I also wouldn't want to move any tabs out of the main window. All 6000+ need to be in the same window, I just need a better way to label them, filter them, search them, organize them, etc. with the end result to slowly start whittling away at that number until I'm under 300 tabs. I'm hoping this addon can help with that but need confirmation it won't force a mass tab reload, or try to shove some tabs onto a separate instance to hide them.

**Comment by Decme** [comment]
I will update this as I learn more about the extension and then convert it into a review once I feel it's in a final state to determine.

* I tested on a separate device, and confirmed it doesn't force tab loads when switching between panels (tab groups).
* It doesn't close tabs either, tabs remain on the top of the page, but you also get localized panels (tab groups) on the sidebar. This is best case scenario, as closing & reopening 6000 tabs each time I swap between panels (tab groups) would have been a deal breaker.
* 'Configure panel' is quite laggy and often prompts Firefox to ask me if I would like to wait for the script to finish etc. but this is likely due to having ~6600 tabs total. Still, if feedback is worth anything, any form of optimization in future releases would go a long way.
* The side bar is adjustable, I've made it a bit wider so that I can see more of the title of the tab. I've noticed right away that I can much more easily spot tabs that I probably don't need any more or can be done with in short order; I suspect this will aid in whittling away at the huge tab count.

* One possible downside I've noticed so far is that the auto group function "Move tabs with matched URLs to this panel" only applies if you freshly load up a tab. This means for example if you make a dedicated Reddit.com panel (tab group), and you have 150 reddit tabs scattered about, they won't automatically move to that panel once the browser restarts.
* If this feature is implemented (or doable), a caution to the developer(s) is that tabs being moved about on the top tab bar as they get re-grouped may need to be rate limited, as I'm unsure how well Firefox would handle moving 150 tabs (or 600 for example) to another part of the tab bar, and also raises the question - does Firefox need to activate a tab to move it? If this is the case it may need to rate limit the number per browser session (with a user defined value & prompt)

I'll add more info as I test, as I'm sure reading this could help other people in future.
